src/Material.cpp

```diff
@@ -1313,7 +1452,7 @@ void Material::checkSigmaT() {
  *          scattering cross-sections and chi for all energy groups.
  * @return character array of this Material's attributes
  */
-std::string Material::toString() {
+std::string MacroMaterial::toString() {
```

Comment:
Perhaps some of the ``MacroMaterial::toString()`` routine should be implemented in the ``Material`` parent class? Since some of the arrays are attributes of ``Material``, it should have its own ``toString()`` routine to print those items. The subclasses can all call the parent class and  then concatenate their own substrings to the end to create a super string.